inference-engine/src/inference_engine/include/ie/ie_parameter.hpp

```diff
@@ -307,12 +306,12 @@ class INFERENCE_ENGINE_API_CLASS(Parameter) {
         }
 
         template <class U>
-        typename std::enable_if<!HasOutputStreamOperator<U>::value, void>::type
-        print(std::ostream& stream, const U& object) const {}
+        typename std::enable_if<!HasOutputStreamOperator<U>::value, void>::type print(std::ostream& stream,
+                                                                                      const U& object) const {}
 
         template <class U>
-        typename std::enable_if<HasOutputStreamOperator<U>::value, void>::type
-        print(std::ostream& stream, const U& object) const {
+        typename std::enable_if<HasOutputStreamOperator<U>::value, void>::type print(std::ostream& stream,
```

Comment:
can we have template and function on separate lines?

Reply:
It happens because here template is equal to return value.
We can have it on a separate line if rewrite this code, or enable`AlwaysBreakAfterReturnType` option, which will affect all functions.
I rewrote these methods.